src/Literature/Literatures.ts

```diff
@@ -1,14 +1,14 @@
 import { CityName } from "./../Enums";
 import { Literature } from "./Literature";
-import { LiteratureNames } from "./data/LiteratureNames";
+import { LiteratureName } from "./data/LiteratureNames";
 import { FactionNames } from "../Faction/data/FactionNames";
 
-export const Literatures: Record<string, Literature> = {};
+export const Literatures = {} as Record<LiteratureName, Literature>;
 
 (function () {
   let title, fn, txt;
   title = "The Beginner's Guide to Hacking";
-  fn = LiteratureNames.HackersStartingHandbook;
+  fn = LiteratureName.HackersStartingHandbook;
```

Comment:
Can I just say, that "fn" to me always means "function" and it is really bugging me when reading this code? XD

(No need to change it though)

Reply:
Yeah that edit I didn't manually look at that code, otherwise I would have changed it because I totally agree. The edit was just from using VSCode to rename the enum as a singular name

Reply:
Went ahead and changed this, as well as the whole structure of Literatures.ts (to match how Programs / Messages initialize their values)